resources/js/components/modules/data/alcohol.js

```diff
@@ -0,0 +1,38 @@
+export const alcohol = {
+  data () {
+    return {
+      slide: {
+        number: '1',
```

Comment:
can you change `number` to `slide_id`

Reply:
also make 'slide_id' value an number type